### PR TITLE
harbor_registry & harbor_replication resources

### PR DESCRIPTION
**Pull request by brett-wright**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,6 +48,18 @@ jobs:
       - goreleaser:
           release-type: << parameters.release-type >>
 
+  acceptance_test:
+    docker:
+      - image: circleci/golang:1.14
```
Comment (peterbrondum) [inline .circleci/config.yml:53]: 1.15 is available if that is of interrest :)

**Review by peterbrondum — COMMENTED** [review]
Otherwise i think it looks good :)

**Review by peterbrondum — APPROVED** [review]

File: docs/resources/registry.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Resource: harbor_registry
+
+
+
+## Example Usage
+
+```hcl
+    resource "harbor_registry" "main" {
+		provider_name = "docker-hub"
```
Comment (lassegaardsholt) [inline docs/resources/registry.md:9]: Make sure the indents are the same, it looks a bit more clean

File: models/replications.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+type ReplicationBody struct {
+	Name        string `json:"name,omitempty"`
+	Description string `json:"description,omitempty"`
+	SrcRegistry struct {
+		ID int `json:"id,omitempty"`
+	} `json:"src_registry,omitempty"`
+	DestRegistry struct {
+		ID int `json:"id,omitempty"`
+	} `json:"dest_registry,omitempty"`
+	DestNamespace interface{} `json:"dest_namespace,omitempty"`
```
Comment (lassegaardsholt) [inline models/replications.go:14]: Shouldn't that be a `string`and not an `interface{}` ?

File: client/replication.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+		}
+		body.Filters = append(body.Filters, filter)
+
+	}
+
+	log.Println(body)
+	return body
+}
+
+// func expandfilters(configured []interface{}) *models.ReplicationFilters {
```
Comment (lassegaardsholt) [inline client/replication.go:69]: If this isn't needed anymore, maybe we should just remove it

**Review by lassegaardsholt — APPROVED** [review]

File: docs/resources/registry.md (hunk 2 above)
Comment (brett-wright) [reply docs/resources/registry.md]: I've fixed formatting

File: models/replications.go (hunk 3 above)
Comment (brett-wright) [reply models/replications.go]: correct. I've changed this

File: client/replication.go (hunk 4 above)
Comment (brett-wright) [reply client/replication.go]: Have removed commented out code

File: .circleci/config.yml (hunk 1 above)
Comment (brett-wright) [reply .circleci/config.yml:53]: updated

**Pull request merged**